newsfragments/44755.significant.rst

```diff
@@ -0,0 +1,26 @@
+Provider configuration fallbacks removed from core
+
+Around ~2.7, we moved provider config from core into the providers themselves.
+However, if providers that were released before that change was used on 2.7+,
+there could be failures because the config wouldn't be in those providers!
+So, we added a fallback for all of the configs that were moved. These fallbacks
+have now been removed. You must use providers that contain their own config.
+
+TODO: It's unlikely this will be the change that sets the "lowest" min provider version
+for compatibility with Airflow 3. But, it is possible, and if it is we will need to
+then determine what those versions are.
```

Comment:
We have that info already in easily parsable way:  

https://github.com/apache/airflow/blob/main/generated/provider_metadata.json

Reply:
Hmm, it's not every provider for 2.7, just those with config. Not going to worry about it right now, I fully expect something else will cause a newer min.